### ISSUE TITLE
warning: non-zero exit status while installing prophet and rstan in Azure notebooks

### ISSUE THREAD

**Issue by harishvenkatesan96** [stan-dev/rstan#684]
Hi, I am trying to install the Prophet package in R Azure Notebooks (in Azure ML Service). I tried various methods for installing the package, but it keeps throwing a warning - “non-zero exit status”. Please find below the exact configuration and steps I followed to install prophet.

I followed  the RStan page (https://github.com/stan-dev/rstan/wiki/RStan-Getting-Started) and configured the C++ toolchain as per the instruction.

![image](https://user-images.githubusercontent.com/35761352/63208838-f1a32700-c0f6-11e9-8a6d-a7fb0dfa7fbb.png)

Then, I tried to install the Prophet package along with its dependencies.

![image](https://user-images.githubusercontent.com/35761352/63208841-01227000-c0f7-11e9-9aaa-9af5c34a0635.png)

But, RStan and Prophet packages were not properly installed and returned a warning- non-zero exit status.

![image](https://user-images.githubusercontent.com/35761352/63208849-15ff0380-c0f7-11e9-9c9a-aa6680ceea84.png)

Can you please help in installing the Prophet package without issues in R Azure Notebooks of Azure ML Service.

Below is the version of my OS and R which might be helpful in debugging the issue.

![image](https://user-images.githubusercontent.com/35761352/63208850-244d1f80-c0f7-11e9-913c-767a7e6ab6e3.png)

**Comment by ellenwterry** [comment]
any progress on this?

**Comment by bgoodri** [comment]
No. Do you have any additional information on the installation error? It might just be not enough RAM available to install RStan and / or prophet from source.
